Mathlib/Combinatorics/SimpleGraph/Maps.lean

```diff
@@ -671,4 +672,17 @@ noncomputable def overFinIso (hc : Fintype.card V = n) : G ≃g G.overFin hc :=
 
 end Finite
 
+section degree
+
+variable (v) [Fintype (G.neighborSet v)] (f : G ≃g G')
+
+instance : Fintype (G'.neighborSet (f v)) :=
+  Fintype.ofEquiv ↑(G.neighborSet v) (f.mapNeighborSet v)
+
+theorem degree_map_eq : G'.degree (f v) = G.degree v := by
```

Comment:
IMHO, this should assume `Fintype` for both `neighborSet`s, so that it works for any instance.

Reply:
Should I remove the instance on line 679 then? Doesn't adding Fintype as an assumption for both `neighborSet`s make the theorem less general?